README.md

```diff
@@ -21,35 +21,29 @@ Documentation can be found [here](https://elfpy.element.fi).
 
 ## Install
 
-Set up your favorite python virutal environment with python == 3.8 (we recommend [pyenv](https://github.com/pyenv/pyenv#how-it-works) and [virtualenv](https://github.com/pyenv/pyenv-virtualenv)). While we don't explicitly support python > 3.8, we haven't had trouble running on later versions (with one exception noted below). For example:
+Set up your favorite python virutal environment with python 3.9 or 3.10 (we recommend [pyenv](https://github.com/pyenv/pyenv#how-it-works) and [virtualenv](https://github.com/pyenv/pyenv-virtualenv)). For example:
 
 ```bash
-pyenv install 3.8.16
-pyenv local 3.8.16
-python -m venv .venv
-source .venv/bin/activate
+pyenv install 3.10
+pyenv local 3.10
+pyenv virtualenv elf-env
```

Comment:
we are calling it `venv` on purpose, keep that syntax

Reply:
oh I noticed we're recommending pyenv-virtualenv then not using it (venv is a different virtual environment manager). was I the one who switched to recommend pyenv-virtualenv?

Reply:
switched text to point to venv, and clarify